contracts/OwnedPermissionManager.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./PermissionInterface.sol";
+
+contract OwnedPermissionManager is PermissionInterface, Ownable {
+    
+    mapping(address => bool) publishers;
+    mapping(address => bool) reviewers;
+    
+    
+    function canPublish(address _address) public view override returns (bool)
+    {
+        return publishers[_address];
+    }
+    
+    function canReview(address _address) public view override returns (bool)
+    {
+        return reviewers[_address];
+    }
```

Comment:
These functions really looks like **function modifiers**, should you use them instead of classic functions ?